src/components/Comment/Comment.js

```diff
@@ -40,16 +47,36 @@ const Date = styled.div`
   font-weight: bold;
 `;
 
-export default function Comment({ data }) {
-  const { creator, content, createdAt } = data;
+const DeleteButton = styled.div`
+  height: 20px;
+  padding-left: 10px;
+  color: red;
+  font-size: 10px;
+  font-weight: bold;
+  cursor: pointer;
+`;
+
+export default function Comment({ data, snippetId, userId }) {
+  const { _id: commentId, creator, content, createdAt } = data;
+
   const formatDate = createdAt.slice(0, 10);
+  const isCreator = creator._id === userId;
+
+  const handleOnclick = async () => {
+    const response = await deleteComment({ userId, commentId, snippetId });
+
+    if (response.result === OK) {
+      alert(DELETE_COMMENT_SUCCEEDED);
+    }
+  };
 
   return (
     <Wrapper>
       <UserImage src={creator.imageUrl} alt="프로필 이미지" width="25px" height="25px" />
       <UserName>{creator.nickname}</UserName>
       <Content>{content}</Content>
       <Date>{formatDate}</Date>
+      {isCreator && <DeleteButton onClick={handleOnclick}>X</DeleteButton>}
```

Comment:
self closing으로 작성할 수 있습니다!

```suggestion
      {isCreator && <DeleteButton onClick={handleOnclick} />}
```